geomstats/learning/exponential_barycenter.py

```diff
@@ -0,0 +1,192 @@
+"""Frechet mean."""
+
+import logging
+
+from sklearn.base import BaseEstimator
+
+import geomstats.backend as gs
+from geomstats.geometry.euclidean import Euclidean
+
+EPSILON = 1e-6
+
+
+def linear_mean(points, weights=None):
```

Comment:
This is the same code as in frechet_mean.py, right? Maybe call that function instead of rewriting it?

Reply:
how about this one?